source/includes/gcp/_disks.md

```diff
@@ -0,0 +1,268 @@
+### Disks
+Deploy and manage your disks.
+
+<!-------------------- LIST DISKS -------------------->
+
+#### List disks
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/disks"
+
+# The above command returns JSON structured like this:
+```
+```json
+{
+  "data": [
+    {
+      "creationTimestamp": "2019-05-07T08:46:06.488-07:00",
+      "sizeGb": "10",
+      "zone": "https://www.googleapis.com/compute/v1/projects/test-area/zones/northamerica-northeast1-a",
+      "status": "ATTACHED",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/zones/northamerica-northeast1-a/disks/d-test-dow",
+      "type": "https://www.googleapis.com/compute/v1/projects/test-area/zones/northamerica-northeast1-a/diskTypes/pd-standard",
+      "lastAttachTimestamp": "2019-05-07T08:46:11.989-07:00",
+      "users": [
+        "https://www.googleapis.com/compute/v1/projects/test-area/zones/northamerica-northeast1-a/instances/i-test-aum"
+      ],
+      "labelFingerprint": "42WmSpB8rSM=",
+      "physicalBlockSizeBytes": "4096",
+      "kind": "compute#disk",
+      "shortUsers": "i-test-aum",
+      "shortType": "pd-standard",
+      "attachMode": "READ_WRITE",
+      "autoDelete": false,
+      "id": "5333546534174463697",
+      "name": "d-test-dow",
+      "shortZone": "northamerica-northeast1-a",
+      "shortRegion": "northamerica-northeast1"
+    }
+  ],
+  "metadata": {
+    "recordCount": 1
+  }
+}
+```
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/disks</code>
+
+Retrieve a list of all disks in a given [environment](#administration-environments)
+
+Attributes | &nbsp;
+------- | -----------
+`creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format
+`sizeGb` <br/>*string* | The size of the disk in GB. Acceptable values are 1 to 65536, inclusive.
+`zone`<br/>*string* | URL of the zone where the instance resides. 
+`status`<br/>*string* | The status of the disk. One of the following values: READY, CREATING, ATTACHING, ATTACHED, DETACHING, RESIZING, DELETING, and DEPRECATED.
+`selfLink`<br/>*string* | Server-defined URL for this resource
+`type`<br/>*string* | URL of the disk type resource describing which disk type to use to create the disk. Choices are ':url/pd-standard' or ':url/pd-ssd'.
+`lastAttachTimestamp`<br/>*string* | Timestamp representing the last time the disk was attached.
+`users`<br/>*Array[String] | Links to the instances the disk is attached to.
+`labelFingerprint`<br/>*string* | A base64-encoded string. A hash of the label's contents and used for optimistic locking
+`physicalBlockSizeBytes`<br/>*string* | Physical block size of the persistent disk, in bytes. Currently supported sizes are 4096 and 16384.
+`kind`<br/>*string* | Type of the resource.
+`shortUsers`<br/>*string* | The names of the instances the disk is attached to.
```

Comment:
Why are instance-related fields called `users`?

Reply:
`users` is the field from GCP, the `shortUsers` is a CloudMC parsed field to indicate that it is the users without the full URI

Reply:
I would like to rename ```shortUsers``` later when we support mounting disks by multiple instances (read only mode only). Maybe something like ```instanceIds``` as that's what it is in the end.